### DISCUSSION TITLE
Network problems

### DISCUSSION THREAD

**Discussion by ghost** [filecoin-project/lotus#4743] [Storage Provider] [answered]
The source version I compiled is ntwk-nerpa-11.04.0，After successful connection to the test network, how should I connect to the main network,           
lotus version 1.1.2+git.3908701b7
The start command is： lotus daemon

**Comment by jennijuju** [comment]
https://docs.filecoin.io/build/local-devnet/#devnet-with-vanilla-lotus-binaries, see the `If you have a running installation of Lotus` section

**Comment by ghost** [comment]
Do I have to use a snapshot now if I want my nodes to connect to the main network?

**Comment by ghost** [comment]
@jennijuju

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
get the latest release. build it and then import a snapshot. 

https://docs.filecoin.io/get-started/lotus/installation/#build-and-install-lotus

the nerpa binaries won't connect to the mainnet
